mlir/lib/Dialect/AffineOps/AffineOps.cpp

```diff
@@ -134,8 +134,9 @@ bool mlir::isValidDim(Value value) {
     return false;
   }
   // This value has to be a block argument for a FuncOp or an affine.for.
-  auto *parentOp = value.cast<BlockArgument>()->getOwner()->getParentOp();
-  return isa<FuncOp>(parentOp) || isa<AffineForOp>(parentOp);
+  // auto *parentOp = value.cast<BlockArgument>()->getOwner()->getParentOp();
```

Comment:
I presume this accidentally snuck in?

Reply:
Yeah, I'll clean this up once it's ready. I'm still having issues with the custom parser.